nixos/tests/mutable-users.nix

```diff
@@ -63,7 +63,7 @@ import ./make-test-python.nix ({ pkgs, ...} : {
             expected_hashes[file] = machine.succeed(f"sha256sum {file}")
             expected_stats[file] = machine.succeed(f"stat {file}")
 
-        machine.succeed("/run/current-system/bin/switch-to-configuration dry-activate")
+        machine.succeed("/run/booted-system/specialisation/mutable/bin/switch-to-configuration dry-activate")
```

Comment:
Is this correct? It wasn’t using `${mutableSystem}` before.

Reply:
Yes, because /run/current-system at that point is the specialisation, this just makes it more explicit.